crates/python/src/register_data.rs

```diff
@@ -0,0 +1,15 @@
+use pyo3::{
+    types::{PyComplex, PyFloat, PyInt},
+    Py,
+};
+use qcs::RegisterData;
+use rigetti_pyo3::py_wrap_union_enum;
+
+py_wrap_union_enum! {
+    PyRegisterData(RegisterData) as "RegisterData" {
+        i8: I8 => Vec<Vec<Py<PyInt>>>,
+        f64: F64 => Vec<Vec<Py<PyFloat>>>,
+        i16: I16 => Vec<Vec<Py<PyInt>>>,
+        complex32: Complex32 => Vec<Vec<Py<PyComplex>>>
```

Comment:
Does this `complex32` have to agree with the one in [crates/lib/src/api.rs#Register](https://github.com/rigetti/qcs-sdk-rust/pull/230/files#diff-0444195b31be2fc4d234f4a0834b34600c41e782bee4e376391658cb369a1987L216) ?

Reply:
Not sure what you mean. The `Complex32` inner type in both Rust types is the same, just the variant name is different.

Reply:
As you say, something that would need to be addressed upstream, so IMO resolved in terms of this MR.